.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Vald",
+  "build": {
+    "dockerfile": "../dockers/dev/Dockerfile"
+  },
+  "runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash",
+    "go.useGoProxyToCheckForToolUpdates": false,
+    "go.autocompleteUnimportedPackages": true,
+    "go.buildOnSave": true,
+    "go.vetOnSave": true,
+    "go.lintOnSave": true,
+    "go.testOnSave": true,
+    "go.gopath": "/go"
+  },
+  "extensions": [
```

Comment:
@kmrmt Do you have any recommends of extensions for Go development in VSCode?

Reply:
LGTM :+1: